cmd/fleetctl/query.go

```diff
@@ -138,6 +139,11 @@ func queryCommand() *cli.Command {
 				if strings.Contains(err.Error(), "no hosts targeted") {
 					return errors.New(fleet.NoHostsTargetedErrMsg)
 				}
+				if strings.Contains(err.Error(), fleet.InvalidLabelSpecifiedErrMsg) {
+					regex := regexp.MustCompile(`(\[.*?\])`)
+					match := regex.FindString(err.Error())
+					return errors.New(fmt.Sprintf("%s %s", match, fleet.InvalidLabelSpecifiedErrMsg))
+				}
```

Comment:
This complex logic is needed because `server/service/base_client.go#parseResponse`https://github.com/fleetdm/fleet/blob/main/server/service/base_client.go#L64 formats the error like this:
```
create live query: POST /api/latest/fleet/queries/run_by_identifiers received status 400 Bad request: [iamnotalabel] are not valid label names, remove to continue
```

Reply:
We should be able to unwrap the error to make this easier, see https://pkg.go.dev/fmt#Errorf

Reply:
sorry, not sure what you mean by "unwrap the error"? But I changed it such that the linting error is gone.

Reply:
See https://pkg.go.dev/errors re: error wrapping.